api/v1beta1/chaincode.go

```diff
@@ -0,0 +1,33 @@
+package v1beta1
+
+const (
+	// ChaincodeProposalLabel the main purpose of this label is to make it easier to clean up the proposal when the chaincode is deleted.
+	ChaincodeProposalLabel = "bestchains.chaincode.delete.proposal"
```

Comment:
better to add more labels:
- bestchains.chaincode.channel: 'xxx'
- bestchains.chaincode.id: ''..."  
- bestchains.chaincode.version: ''..."